src/components/Calculator.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import calculate from '../logic/calculate';
+import './calculator.css'
```

Comment:
- [ ] @a-laarabi great work on the design , it would be much better if you tried to make the calculator fit in the page, currently I have to use the scroll feature to access the whole calculator:
![image](https://user-images.githubusercontent.com/61408860/185438797-233bbb95-5cc3-4fe4-8ae5-67e1d1418cae.png)